src/org/jitsi/impl/neomedia/rtp/StreamRTPManager.java

```diff
@@ -168,14 +171,17 @@ public void putResumableStreamRewriter(
     public ResumableStreamRewriter getResumableStreamRewriter(Long ssrc,
                                                               boolean create)
     {
-        ResumableStreamRewriter rewriter = ssrcToRewriter.get(ssrc);
-        if (rewriter == null && create)
+        synchronized (ssrcToRewriter)
```

Comment:
Personally, I am more worried about someone wasting person time debugging a race condition months from now when the code has changed, than about the unknown gain in performance. But I'll remove the commit.